src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -295,7 +295,7 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 			},
 			'workbench.editor.enablePreview': {
 				'type': 'boolean',
-				'description': localize('enablePreview', "Controls whether opened editors show as preview editors. Preview editors do not stay open, are reused until explicitly set to be kept open (via double-click or editing), and show file names in italics."),
+				'description': localize('enablePreview', "Controls whether preview mode is used when documents open. There is a maximum of one preview mode editor per editor group. This editor displays its filename in italics on its tab or title label and in the Open Editors view. Its contents will be replaced by the next document opened in preview mode. Editing a preview mode document will persist it, as will a double-click on its label, or the 'Keep Open' option in its label context menu. Opening a document from Explorer with a double-click persists it immediately."),
```

Comment:
I would keep using "editor" and not "document" because this really applies to any kind of editor, even non-document ones.